docs/reference/support-duration.md

```diff
@@ -13,6 +12,6 @@ for Ubuntu and open source](https://ubuntu.com/security/esm) for further details
 | Ubuntu Core 18 | Jan 2019 | Jan 2029 | Jun 2030 | amd64, arm64, i386, armhf | 2.52 | 7.x |  |
 | Ubuntu Core 20 | Feb 2021 | Feb 2031 | Jun 2032 | amd64, arm64, armhf | 2.58 | 8.x |  |
 | Ubuntu Core 22 | Jun 2022 | Jun 2032 | Jun 2034 | amd64, arm64, armhf | 2.59 | 8.x |  |
-| Ubuntu Core 24 | Jun 2024 | Jun 2034 | Jun 2036 | amd64, arm64 | latest | 8.x |  |
+| Ubuntu Core 24 | Jun 2024 | Jun 2034 | Jun 2036 | amd64, arm64, armhf | latest | 8.x |  |
 
-See [Testing platforms](testing-platforms) for image download links, and for more details on Snapcraft support, see [Snapcraft bases](https://documentation.ubuntu.com/snapcraft/stable/reference/bases/).
+See [Testing platforms](testing-platforms) for download links to device-specific images that can help with image building, and for more details on Snapcraft support, see [Snapcraft bases](https://documentation.ubuntu.com/snapcraft/stable/reference/bases/).
```

Comment:
I'm not sure how preexisting images connect to helping with making your own images

Reply:
Thanks - I'll try to tweak this. It was a clumsy attempt to link testing images with our messaging around building your own image.